libs/execution_context/ebpf_native.c

```diff
@@ -398,7 +428,7 @@ _ebpf_native_release_reference_internal(void* base_object, ebpf_file_id_t file_i
 
 static NTSTATUS
 _ebpf_native_provider_attach_client_callback(
-    HANDLE nmr_binding_handle,
+    _In_ HANDLE nmr_binding_handle,
```

Comment:
FYI, whether `_In_` belongs on HANDLE or not depends on whether you view HANDLE as a uint64_t or as a `void*`. I.e., is it an opaque value that may or may not be a pointer, or is it specifically a pointer.  `_In_` belongs if it is specifically a pointer and does not belong if it is an opaque value that should not be assumed to be a pointer.

Reply:
The annotation is defined in the public headers and can't be modified.